lightly/utils/bounding_box.py

```diff
@@ -77,24 +80,49 @@ def clip_to_0_1(value: float) -> float:
 
     @classmethod
     def from_x_y_w_h(cls, x: float, y: float, w: float, h: float) -> BoundingBox:
-        """Helper to convert from bounding box format with width and height.
+        """Creates a BoundingBox from x, y, width, and height.
+
+        Args:
+            x: 
+                x coordinate of the top-left corner.
+            y: 
+                y coordinate of the top-left corner.
+            w: 
+                Width of the bounding box.
+            h: 
+                Height of the bounding box.
+
+        Returns:
+            BoundingBox: A BoundingBox instance.
 
         Examples:
             >>> bbox = BoundingBox.from_x_y_w_h(0.1, 0.2, 0.2, 0.2)
-
+        
         """
         return cls(x, y, x + w, y + h)
 
     @classmethod
     def from_yolo_label(
         cls, x_center: float, y_center: float, w: float, h: float
     ) -> BoundingBox:
-        """Helper to convert from yolo label format
-        x_center, y_center, w, h --> x0, y0, x1, y1
+        """Creates a BoundingBox from YOLO label format.
 
-        Examples:
-            >>> bbox = BoundingBox.from_yolo(0.5, 0.4, 0.2, 0.3)
+        Args:
+            x_center: 
+                x coordinate of the center.
```

Comment:
Again: Let's document that the values are relative to the image width/height and not in the unit of pixels.
```suggestion
                x coordinate of the center.  Relative to the image width.
```